keps/sig-api-machinery/4006-transition-spdy-to-websockets/README.md

```diff
@@ -417,14 +414,18 @@ affect the perceived performance.
 3. As releases increment, the probablity of a WebSocket enabled `kubectl` communicating
 with an older non-WebSocket enabled API Server decreases.
 
-### Proposal: `K8s-Websocket-Protocol: stream-translate` Header
+### Proposal: New `RemoteCommand` Sub-Protocol Version - `v5.channel.k8s.io`
```

Comment:
Would it make sense mentioning that this protocol is enhanced version of v4.channel.k8s.io, hence it already includes half-close, etc.?

Reply:
Depending on the specifics of the expanded stream ID space, it won't be exactly like v4